server/src/main/java/com/datastrato/gravitino/server/web/rest/ExceptionHandlers.java

```diff
@@ -43,6 +49,39 @@ public static Response handleMetalakeException(OperationType op, String metalake
     return MetalakeExceptionHandler.INSTANCE.handle(op, metalake, "", e);
   }
 
+  private static class PartitionExceptionHandler extends BaseExceptionHandler {
+
+    private static final ExceptionHandler INSTANCE = new PartitionExceptionHandler();
+
+    private static final String PARTITION_MSG_TEMPLATE =
+        "Failed to operate partition(s)%s operation [%s] of table [%s], reason [%s]";
+
+    @Override
+    public Response handle(OperationType op, String partition, String table, Exception e) {
+      String formatted = StringUtil.isBlank(partition) ? "" : " [" + partition + "]";
+      String errorMsg =
+          String.format(
+              PARTITION_MSG_TEMPLATE, formatted, op.name(), table, e.getClass().getSimpleName());
```

Comment:
`e.getClass().getSimpleName()` is not correct. You need to rebase the latest code and use `getErrorMsg`.

Reply:
fixed